composer.json

```diff
@@ -22,10 +22,12 @@
   },
   "scripts": {
     "post-install-cmd": [
-      "\"vendor/bin/phpcs\" --config-set installed_paths vendor/wp-coding-standards/wpcs"
+      "\"vendor/bin/phpcs\" --config-set installed_paths vendor/wp-coding-standards/wpcs",
+      "npm install"
     ],
     "post-update-cmd": [
-      "\"vendor/bin/phpcs\" --config-set installed_paths vendor/wp-coding-standards/wpcs"
+      "\"vendor/bin/phpcs\" --config-set installed_paths vendor/wp-coding-standards/wpcs",
+      "npm install"
```

Comment:
Let's remove the second `npm install`. I don't think we want to trigger `npm install` every time a developer runs `composer update`. The goal should be to streamline the initial setup of Give in a local environment, so it makes sense to hook into `composer install` but not `composer update`.

Reply:
@kevinwhoffman The first one `post-install-cmd` will work only with a lock file present, which won't be the case in the initial setup.

`post-update-cmd` is for both update or install _without_ the lock file present, which will be the scenario after cloning the repo.

Reply:
@Sidsector9 A `composer.lock` file should be committed to the Git repo. I created an issue for it in https://github.com/WordImpress/Give/issues/2896. Please make sure `composer.lock` gets committed and then continue with my recommendation above to remove the second mention of `npm install`.

Reply:
@kevinwhoffman , I'll do that :+1: